backend/src/main/java/se/chalmers/dat265/group1/storage/PostgresSQLConnector.java

```diff
@@ -45,7 +43,9 @@ public ResultSet executeQuerry(String query) {
             stmt = this.conn.prepareStatement(query);
             return stmt.executeQuery();
         } catch (SQLException e) {
-            e.printStackTrace();
+            log.error("executeQuerry", e);
+        } finally {
+            System.out.println("Execute Querry failed");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)